packages/framework-core/src/booster-read-models-reader.ts

```diff
@@ -60,6 +60,10 @@ export class BoosterReadModelsReader {
     if (readModelRequest.filters) {
       searcher.filter(readModelRequest.filters)
     }
+    searcher
+      .limit(readModelRequest.limit)
+      .afterCursor(readModelRequest.afterCursor)
+      .paginatedVersion(readModelRequest.paginatedVersion)
```

Comment:
what's the `paginatedVersion` for?

Reply:
Is just a temporal property to support paginated and not paginated Read Models. In the future, we will deprecate the non-paginated and remove it eventually, so it won't be necessary.